bundle-workflow/tests/resources/build_manifest.yml

```diff
@@ -0,0 +1,2462 @@
+build:
```

Comment:
1. The file should include its version, to avoid future confusion, this is a 1.1.0 buld manifest.
2. We've started a pattern of putting these files under a `data` subfolder of the tests that use them, e.g. https://github.com/opensearch-project/opensearch-build/tree/main/bundle-workflow/tests/test_manifests/data, rather than having a bucket for resources, so we don't end up with one test modifying other tests' data.

Reply:
these were just place holders for referencing until we start reading manifest files from s3. Removed them